source/crud/crud.rst

```diff
@@ -336,165 +338,209 @@ Read
+     *
+     * Note: This option is specified as "maxTimeMS" in the getMore command and not provided as part of the 
+     * initial find command.
      *
-     * On servers < 3.2, this option is ignored.
+     * @see https://docs.mongodb.com/manual/reference/command/find/
      */
     maxAwaitTimeMS: Optional<Int64>;
 
     /**
-     * The maximum amount of time to allow the query to run. If $maxTimeMS also exists
-     * in the modifiers document, the maxTimeMS field overwrites $maxTimeMS.
+     * Maximum number of documents or index keys to scan when executing the query.
      *
-     * The default is no maxTimeMS.
+     * This option is sent only if the caller explicitly provides a value. The default is to not send a value.
      *
-     * @see http://docs.mongodb.org/manual/reference/operator/meta/maxTimeMS/
+     * @see https://docs.mongodb.com/manual/reference/method/cursor.max/#cursor.max
+     */
+    maxScan: Optional<Int64>;
+
+    /**
+     * The maximum amount of time to allow the query to run.
+     *
+     * This option is sent only if the caller explicitly provides a value. The default is to not send a value.
+     *
+     * @see https://docs.mongodb.com/manual/reference/command/find/
      */
     maxTimeMS: Optional<Int64>;
 
     /**
-     * Meta-operators modifying the output or behavior of a query.
-     * The default is no modifers.
+     * The inclusive lower bound for a specific index.
      *
-     * @see http://docs.mongodb.org/manual/reference/operator/query-modifier/
+     * This option is sent only if the caller explicitly provides a value. The default is to not send a value.
+     *
+     * @see https://docs.mongodb.com/manual/reference/method/cursor.min/#cursor.min
      */
-    modifiers: Optional<Document>;
+    min: Optional<Document>;
 
     /**
      * The server normally times out idle cursors after an inactivity period (10 minutes) 
      * to prevent excess memory use. Set this option to prevent that.
      *
-     * The default in servers >= 3.2 is no value: no "noCursorTimeout" option is sent with
-     * the "find" command.
-     *
-     * In servers < 3.2, the NoCursorTimeout OP_QUERY flag defaults to false.
+     * This option is sent only if the caller explicitly provides a value. The default is to not send a value.
+     * For servers < 3.2, the NoCursorTimeout wire protocol flag defaults to false.
      *
-     * @see http://docs.mongodb.org/meta-driver/latest/legacy/mongodb-wire-protocol/#op-query
+     * @see https://docs.mongodb.com/manual/reference/command/find/
      */
     noCursorTimeout: Optional<Boolean>;
 
     /**
      * Internal replication use only - driver should not set
      *
-     * The default in servers >= 3.2 is no value: no "oplogReplay" option is sent with
-     * the "find" command.
+     * This option is sent only if the caller explicitly provides a value. The default is to not send a value.
+     * For servers < 3.2, the OplogReplay wire protocol flag defaults to false.
```

Comment:
Same as my comment for `allowPartialResults`. Should we explicitly say that this sets the OplogReplay wire protocol flag?